### DISCUSSION TITLE
Open of file libdeepmd_lmp.so failed

### DISCUSSION THREAD

**Discussion by theAfish** [deepmodeling/deepmd-kit#1672] [Q&A]
Hi, I today installed LAMMPS (plugin mode) following https://github.com/deepmodeling/deepmd-kit/blob/master/doc/install/install-lammps.md. The installation went smoothly and lmp -h works. However, I cannot find the pair style deepmd with the "lmp -h". And when I tried running a lammps job with deep potential, it failed and showed that : 
```
LAMMPS (29 Sep 2021 - Update 3)
  using 1 OpenMP thread(s) per MPI task
plugin load libdeepmd_lmp.so
Open of file libdeepmd_lmp.so failed: libdeepmd_lmp.so: cannot open shared object file: No such file or directory

...

ERROR: Unrecognized pair style 'deepmd' (src/force.cpp:278)
Last command: pair_style      deepmd ../80w.pb
```
Moreover, I checked the source dir of deepmd and cannot find the "libdeepmd_lmp.so" file.

I am wondering how to deal with this issue?

**Comment by njzjz** [comment]
Please take a look at [install-from-source](https://github.com/deepmodeling/deepmd-kit/blob/v2.1.1/doc/install/install-from-source.md). You need to set `-DLAMMPS_SOURCE_ROOT=<value>` when you install the C++ interface.

**Reply by theAfish** [reply]
Thank you! I'll take a look at it.
